.github/workflows/ci.yml

```diff
@@ -63,3 +67,25 @@ jobs:
       with:
         name: built-docs
         path: docs/build/html
+
+  amend:
+      name: "Add PR links to staged changelogs"
+      runs-on: ubuntu-latest
+      needs:
+        - build
+        - build-docs
+      steps:
+          - uses: actions/checkout@v4
+            with:
+              ref: ${{ github.head_ref }}
+
+          - uses: actions/setup-python@v5
+            with:
+              python-version: "3.13"
+
+          - name: "Amend and push any staged changelogs"
+            run: |
+              git config user.name "github-actions[bot]"
+              git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
This is just the standard github actions bot. We could use something else if we want